### DISCUSSION TITLE
What does the option "Remove successful installs/uninstalls/updates from the installation list" do?

### DISCUSSION THREAD

**Discussion by ioqy** [Devolutions/UniGetUI#685] [General]
I would expect that it would not show already installed packages in the `Discover Packages` list, but every package that is already installed is still in the list (winget and chocolatey). Is it expected behaviour that already installed packages are shown in the `Discover Packages` list? And is it also expected behaviour that I can install the same package multiple times from `Discover Packages`?

**Comment by marticliment** [comment]
It removes them from the bottom of the window, where all running operations/jobs are listed

**Reply by ioqy** [reply]
Thanks for the answer.

Do you maybe also have an answer to my other two questions?

**Comment by marticliment** [comment]
> I would expect that it would not show already installed packages in the Discover Packages list, but every package that is already installed is still in the list (winget and chocolatey). Is it expected behaviour that already installed packages are shown in the Discover Packages list?

Yes, since this list shows all available packages through the package managers you have selected

> And is it also expected behaviour that I can install the same package multiple times from Discover Packages?

Yes. You might need to reinstall a program for whatever the reason, or you might just want to see more info about an already-installed package

**Comment by ioqy** [comment]
So if I'm in the discover packages list there no way of knowing if I have a package already installed, but to go to the installed packages list and search for the same package there?

**Reply by marticliment** [reply]
exactly
